layout/doc/layout.md

```diff
@@ -1,3 +1,8 @@
+```js script
+import '@api-viewer/docs';
```

Comment:
Should be declared in the mdjs.config.js ?

Reply:
actually yes, not a good idea to important in each file, will change, thanks!

Reply:
fixed